src/pages/auth/LoginPage/KakaoLoginAccessPage.tsx

```
@@ -22,56 +24,36 @@ function KakaoLoginAccessPage() {
-                    localStorage.setItem('loginCheck', 'false');
-                    debugger;
-                    navigate('/main', { replace: true });
-                } catch (error) {
-                    const err = error as AxiosError;
-                    if (err.response?.status === 409) {
-                        console.warn('로그인 실패, 신규 회원 처리', error);
+                debugger;
+                navigate('/main', { replace: true });
+            } catch (error) {
+                const err = error as AxiosError;
+                if (err.response?.status === 409) {
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)